cmd/av/stack_split.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"emperror.dev/errors"
+	"github.com/aviator-co/av/internal/git"
+	"github.com/aviator-co/av/internal/utils/colors"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/spf13/cobra"
+)
+
+var stackSplitCmd = &cobra.Command{
```

Comment:
We merged in our other changes, and we've essentially deprecated adding commands under `av stack <command>`. I think it would make a lot of sense to implement this as a flag under the `av branch` command, probably `av branch --split` which without an argument generates a branch name based on commit message, and if you pass in `av branch --split <name>` creates a branch with the supplied name.